src/App.js

```diff
@@ -18,6 +19,7 @@ function App() {
           <Route path='/dashboard' component={Dashboard} />
           <Route path='/login' component={Login} />
           <Route path='/tables' component={Tables} />
+          <Route path='/project-detail' component={ProjectDetail} />
```

Comment:
I think it's better to have dynamic route smth as 'admin-panel.com/projects/:id' where :id is Project id

Reply:
Postponed https://trello.com/c/BgfSPOet

Reply:
https://trello.com/c/fpnoYKTU

Reply:
```
<Route exact path='/projects' component={Project} />
<Route path='/projects/:id' component={ProjectDetail} />
```
Thanks, I updated route!